demo.md

```diff
@@ -0,0 +1,53 @@
+# Akri and Krustlet integration demo
+This is a demo to showcase the usage of a wasi discovery handler with container Akri's components.
+
+## Start your kubernetes cluster
+
+For this demo we are using microk8s, but feel free to use any of your choice, krustlet has documentation for most of them.
+
+## Start your krustlet node
+
+This can be done following krustlet tutorials [here](https://github.com/deislabs/krustlet/blob/main/docs/intro/quickstart.md).
+
+## Start Akri
+
+Lets make our cluster ready for handleling the discovered devices and for this we are using Akri's Debug Echo configurations, more information can be found [here](https://github.com/deislabs/akri/blob/main/docs/debug-echo-configuration.md).
+
+```
+helm repo add akri-helm-charts https://deislabs.github.io/akri/
+helm install akri akri-helm-charts/akri \
+    --set agent.allowDebugEcho=true \
+    --set debugEcho.discovery.enabled=true \
+    --set debugEcho.discovery.image.repository=ghcr.io/rodz/wde-grpc-proxy-discovery-handler \
+    --set debugEcho.discovery.image.tag=v1 \
+    --set debugEcho.configuration.enabled=true \
+    --set debugEcho.configuration.shared=false
+```
+
+## Apply gRPC proxy configuration
+
+If you log the debug echo pod you should see some errors, that’s because we haven't set up the correct volume mount. Since we are using files to communicate with the Wasi Debug Echo we need to make sure they are sharing the same storage on the machine.
```

Comment:
So erroring is expected behavior? Should the errors be removed/changed then? Are the errors the result of order of deployment?

Reply:
Yes, the error is the correct behavior, this is because we cant add a new volume mount to the daemonset deployment during helm installation of akri, this needs to be done manually.

The order of deployment should not have a difference here. I guess I can create a issue on akri so new volumes could be added by default.

Reply:
You could also update the helm Values.yaml and debug-echo-discovery-handler.yaml files in this repository, right? I'm not sure that is an issue Akri for this makes sense as the use case is specific to WASM currently. I think we'd need more scenarios to make it worth adding the helm values and updating the charts